charts/intel-gpu-resource-driver/values.yaml

```
@@ -19,6 +19,24 @@ serviceAccount:
 
 kubeletPlugin:
   podAnnotations: {}
-  tolerations: []
-  nodeSelector: {}
+  nodeSelector:
+    intel.feature.node.kubernetes.io/gpu: "true"
```

This label will exist only with NFD.   I think it's better to wrap it inside `nfd.enabled` check in the `DaemonSet` template, and remove or leave this empty (in case user wants for some reason to use his/her own label instead of NFD one):
```suggestion
    # label used when nfd.enabled is true
    #intel.feature.node.kubernetes.io/gpu: "true"
```